src/app/core/integration/integration-data.ts

```diff
@@ -0,0 +1,12 @@
+import { PageInfo } from '../shared/page-info.model';
+import { IntegrationModel } from './models/integration.model';
+
+/**
+ * A class to represent the data retrieved by a Eperson service
```

Comment:
An EPerson service? 
Could you give another example, besides an authority value, of something that could be an integration (in the future perhaps)? Because it isn't entirely clear to me what they're supposed to represent.

Reply:
I supposed to create an abstract service like the `ConfigService`.
If we think that there will not be other integration endpoints I remove it and use only `AuthorityService`